vulnerabilities/cve-2022-2385.yaml

```diff
@@ -0,0 +1,32 @@
+title: AWS IAM Authenticator for Kubernetes AccessKeyID Validation Bypass
+slug: CVE-2022-2385
+cves:
+- CVE-2022-2385
+affectedPlatforms:
+- AWS
+affectedServices:
+- EKS
+image: https://cdn.pixabay.com/photo/2019/09/25/09/50/biometrics-4503187_960_720.jpg
+severity: high
+discoveredBy:
+  name: Gafnit Amiga
+  org: Lightspin
+  domain: https://www.lightspin.io/
+  twitter: gafnitav
+disclosedAt: 2022/05/25
```

Comment:
@korniko98 Is this the right date? The blog post and AWS security bulletin have today's date

Reply:
Oooh, this is the date of disclosure to AWS, not the public disclosure. Got it.

Reply:
@patricksanders we should really add another field to the YAML format for publication date though - I'll list a new issue for that